### PR TITLE
Fix typos issues

### PR DESCRIPTION
**Pull request by BigPeep0doo**
### Description of the changes

Fix trivia typos

### Have the changes in this PR been tested?

Yes

### CODE REVIEW

**Comment by BigPeep0doo** [comment]
I've find out that Issue #5900 (Trivia files: fix typos) is outdated so i've created a new one.

File: redbot/cogs/trivia/data/lists/entertainment.yaml [hunk 1]
```diff
@@ -458,10 +458,10 @@ This band's highly original video for "Whip it," characterized by red flower pot
 - Devo
 This electronic instrument's creator was surnamed Moog, and his models are worth a fortune! Other brands include Roland, Korg, and Casio?:
 - Synthesizer
-This female artist enjoyed sucess on both popular and country & western stations with such tunes as "Let Me Be There" and "Have You Never Been Mellow"?:
+This female artist enjoyed success on both popular and country & western stations with such tunes as "Let Me Be There" and "Have You Never Been Mellow"?:
 - Olivia Newton-John
 - olivia newton john
-This film starring Julie Andrews and Christopher Plummer wont he best picture Oscar for 1965?:
+This film starring Julie Andrews and Christopher Plummer won't he best picture Oscar for 1965?:
```
Comment (Flame442) [inline redbot/cogs/trivia/data/lists/entertainment.yaml]:
```suggestion
This film starring Julie Andrews and Christopher Plummer won the best picture Oscar for 1965?:
```

File: redbot/cogs/trivia/data/lists/overwatch.yaml [hunk 2]
```diff
@@ -161,7 +161,7 @@ On which map was this screenshot taken? http://i.imgur.com/vxqkKQ0.jpg:
 On which map was this screenshot taken? http://i.imgur.com/xuO8PDw.jpg:
 - Volskaya Industries
 - Volskaya
-Orignally when Overwatch was released players could choose to stack multiples of any hero on any team,it was later limited to 1 of each hero per team, and an arcade mode where heros tacking was again possble was added, what is this mode called?:
+Originally when Overwatch was released players could choose to stack multiples of any hero on any team,it was later limited to 1 of each hero per team, and an arcade mode where heroes tacking was again possible was added, what is this mode called?:
```
Comment (Flame442) [inline redbot/cogs/trivia/data/lists/overwatch.yaml]:
```suggestion
Originally when Overwatch was released players could choose to stack multiples of any hero on any team, it was later limited to 1 of each hero per team, and an arcade mode where hero stacking was again possible was added, what is this mode called?:
```

**Review by Flame442 — CHANGES_REQUESTED** [review]
While most of the review comments in the original PR were addressed, a few were not. I've moved those here as well.

Otherwise, this PR looks good, but is missing some of the things that were correctly fixed in the other PR. If you want to add those in this PR as well, I can review them here as well.

**Comment by BigPeep0doo** [comment]
I've added your suggestions to the PR, is it ok?

**Review by Flame442 — APPROVED** [review]
Thanks!

**Pull request merged**